modules/remotecontrol/index.js

```diff
@@ -204,18 +213,31 @@ class RemoteControl {
 
     /**
      * Sends remote control event to the controlled participant.
+     *
      * @param {Object} event the remote control event.
      */
     sendEvent(event) {
         const remoteControlEvent = Object.assign(
-            { name: REMOTE_CONTROL_EVENT_NAME },
+            { name: REMOTE_CONTROL_MESSAGE_NAME },
             event
         );
         this.channel.send({
             method: 'message',
             params: { data: remoteControlEvent }
         });
     }
+
+    /**
+     * Sends a response of a request from Jitsi Meet.
+     *
+     * @param {Object} response the response.
```

Comment:
You don't have to change it if you don't want but seeing a doc that says "response the response" seems unnecessary or at least redundant. A suggestion might be to say what generated the response or what the response will be used for.